auth.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+import dotenv from 'dotenv';
```

Comment:
Better to do this at the earliest stage of your application. (So in the index.js)

Reply:
Yes, you right.